test/urltrack.cpp

```diff
@@ -46,9 +46,9 @@ main(int argc, char** argv)
     char url[128];
 
     for (i = 1; i <= 5; i++) {
-        sprintf(url, "http://server.com/foo/bar%u.html", i);
+        snprintf(url, 128, "http://server.com/foo/bar%u.html", i);
```

Comment:
Might want to memset `url` out beforehand, just in case.

Reply:
Good catch!